english/smores

```diff
@@ -0,0 +1 @@
+You decide to use your lighter and the marshmallows to make smores! It's too bad you have chocolate...
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.